src/main/java/org/gol/Grid.java

```diff
@@ -20,4 +20,23 @@ private void fillGrid() {
             }
         }
     }
+
+    StringBuilder builder = new StringBuilder();
```

Comment:
@Sumlarys the _StringBuilder_ object should be instantiated in the _toString_ method instead of at class level.